packages/shared/src/modules/activity-likes/libs/validation-schemas/activity-like-put.validation-schema.ts

```diff
@@ -0,0 +1,5 @@
+import { z } from "zod";
+
+const activityLikePut = z.object({ activityId: z.number().int() }).required();
```

Comment:
```suggestion
const activityLikeChange = z.object({ activityId: z.number().int() }).required();
```
or something like this